source/install/smtp-email-setup.rst

```diff
@@ -111,6 +111,13 @@ Hotmail
 -  Set **SMTP Port** to **587**
 -  Set **Connection Security** to **STARTTLS**
 
+Office365 / Outlook
+^^^^^^^^^^^^^^^^^^^^^
+
+- Set **SMTP Server Name** to **smtp.office365.com**
```

Comment:
1) Can we make these setting names consistent with the other servers? I'm not sure what "encryption method" is, and how that maps to Mattermost settings. Also, it's missing the username/password fields.
2) Did we confirm we've tested with these settings?